src/test/java/seedu/address/storage/JsonAddressBookStorageTest.java

```diff
@@ -73,7 +73,6 @@ public void readAndSaveAddressBook_allInOrder_success() throws Exception {
         // Modify data, overwrite exiting file, and read back
         original.addPerson(HOON);
 
-        // todo
         // original.removePerson(ALICE);
```

Comment:
Do you think we can uncomment this line with the sample person object `ALICE_P` (which comes from `import static seedu.address.testutil.TypicalPersons.ALICE_P;`). The test should also cover if removed entities are saved in the address book JSON. 

Previously it was removed because we changed some stuff with person descriptor but after personally testing should work okay.